web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+  name: string;
+}
+
+const printLabels = (labels: Record<string, string>) => {
```

Comment:
I know this came from @boyskila but generally I find the prefix `print` confusing for anything that doesn't print anything onto a console or similar. How about `formatLabels`?